scripts/react_native_pods.rb

```diff
@@ -277,8 +277,9 @@ def modify_flags_for_new_architecture(installer, cpp_flags)
 end
 
 def build_codegen!(react_native_path)
-  codegen_repo_path = "#{react_native_path}/packages/react-native-codegen";
-  codegen_npm_path = "#{react_native_path}/../react-native-codegen";
+  relative_installation_root = Pod::Config.instance.installation_root.relative_path_from(Pathname.pwd)
+  codegen_repo_path = "#{relative_installation_root}/#{react_native_path}/packages/react-native-codegen";
```

Comment:
Hi @danilobuerger! Is `react_native_path` guaranteed to be relative? Does this work if it is absolute?

Reply:
@dmitryrykun guaranteed might be a bit too strong 😅 But currently its always returned as a relative path from `use_native_modules!`. See https://github.com/react-native-community/cli/blob/master/packages/platform-ios/native_modules.rb#L124 

Also the rest of the code assumes its relative too: For example https://github.com/facebook/react-native/blob/2a5265dff7e654f57b43335804840692313f2a56/scripts/react_native_pods.rb#L429